components/brave_vpn/brave_vpn_service_desktop.cc

```diff
@@ -105,6 +125,9 @@ void BraveVpnServiceDesktop::RemoveVPNConnnection() {
 }
 
 void BraveVpnServiceDesktop::Connect() {
+  if (state_ == ConnectionState::CONNECTING)
```

Comment:
Should this maybe only try connecting if you're in disconnected status? Just thinking about if you call connect twice or if it's called while disconnecting

Reply:
I added this check to avoid calling `Connect()` sequentially while previous connect request is in-progress.
I thought calling this again from any other status except `Connecting` seems fine.